docs/builders/googlecompute.mdx

```diff
@@ -521,3 +521,20 @@ CentOS and recent Debian images have root ssh access disabled by default. Set
 
 The machine type must have a scratch disk, which means you can't use an
 `f1-micro` or `g1-small` to build images.
+
+## Extra disk attachments
+
+@include 'builder/googlecompute/BlockDevice.mdx'
+
+These can be defined using the [disk_attachment](#disk_attachment) block in the configuration.
+
+Note that this is an array, and therefore in HCL2 can be defined as multiple blocks, each
+one corresponding to a disk that will be attached to the instance you are booting.
```

Comment:
Would it be helpful to add an example hcl2 block here?

Reply:
Done, I added one example with two disks defined in the template. Also to make sure it works well, I added an acceptance test for this case.